exampleSite/config.toml

```diff
@@ -1,8 +1,8 @@
 baseURL = "https://www.example.com"
 languageCode = "en-us"
 title = "My Digital Garden"
-theme = "hugo-digital-garden-theme"
-themesdir = "../../"
+theme = "digital-garden"
```

Comment:
Hi dude, I suggest leaving the theme name the same as the project name. It will be better for beginners. Because if we simply use the git submodule, the directory's default name will be the same as the project name on Github.

Reply:
Yes, I agree, we'd better update the setup docs. If we just use the command: `git submodule add https://github.com/paulmartins/hugo-digital-garden-theme.git` then the dir will be `hugo-digital-garden-theme`. To keep the project name the same as the package's is a good experience.